apps/nextjs-website/react-components/types/HighlightBox/HighlightBox.types.ts

```diff
@@ -0,0 +1,9 @@
+export interface HighlightBoxProps {
+  title: string;
+  body: string;
+  imageUrl: string | { data: { attributes: { url: string } } };
```

Comment:
```suggestion
  imageURL: string;
```
Avoid options whenever possible.
Also, write URL in all caps like everywhere else.